clients/go/gengo.go

```diff
@@ -408,27 +409,18 @@ func shortHash(s string) string {
 }
 `
 
-func getVersionSuffix(version string) string {
```

Comment:
No longer needed as I'm using a function that extracts more than that from the packageName and outputPaths